PINRemoteImage.podspec

```diff
@@ -76,7 +76,9 @@ Pod::Spec.new do |s|
   s.subspec "PINCache" do |pc|
     pc.dependency 'PINRemoteImage/Core'    
   	pc.dependency 'PINCache', '=3.0.1-beta.6'
-	pc.osx.deployment_target = osx_deployment
+    pc.ios.deployment_target = ios_deployment
```

Comment:
Super minor nit: mind fixing the spacing?

Reply:
no problem, will fix tomorrow morning. Thank you for review